package.json

```diff
@@ -9,8 +9,8 @@
     "build:mainnet": "node build/build.js --network mainnet",
     "build:devnet": "node build/build.js --network devnet",
```

Comment:
This should probably also be updated to use `--env.network` as you removed minimist.

Reply:
We can't use `--env.network` directly here, because it's a webpack thing I think. I didn't remove `minimist` it's in a central file now: `arugment-parser.js .
Same for other comments.

Reply:
Ah yeah, overlooked that on mobile. Will test the changes tomorrow morning when I am back.